### DISCUSSION TITLE
SimpleStorage - brownie

### DISCUSSION THREAD

**Discussion by mohsin5432** [smartcontractkit/full-blockchain-solidity-course-py#94] [Q&A] [answered]
anyone tried to deploy name to favourite number function in brownie ?

**Comment by mohsin5432** [comment, marked as answer]
def nameToReg():
    account = get_account()
    simple_storage = SimpleStorage.deploy({"from": account})
    name = input("Enter your name?")
    regno = input("Enter your reg no ?")
    transaction = simple_storage.addPerson(name, regno, {"from": account})
    transaction.wait(1)
    regnoc = input("Enter Regno to check user name ?")
    updated_stored_value = simple_storage.RegnotoName(regnoc)
    print(updated_stored_value)


thanks i tried and it worked :)  + i modified that contract to save user name and registration number , then we can retrieve user name through registration number

**Reply by PatrickAlphaC** [reply]
niceeee
